GameData/ROCapsules/PartConfigs/Apollo/ApolloHGA.cfg

```diff
@@ -7,9 +7,9 @@ PART
 	RSSROConfig = true
 	RP0conf = true
```

Comment:
Don't set RP0 configuration state.  Also: only change to this file was whitespace